custom_typings/dom.d.ts

```diff
@@ -0,0 +1,11 @@
+declare interface Window {
+  customElements: CustomElementRegistry;
```

Comment:
Definitely, and these typings are anemic too. Since they're part of the DOM spec, I think they should just go into "dom".